src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -184,21 +184,21 @@ const NotesSection = ({ selectedRelease }) => {
         You can find the{' '}
         <InlineLink
           href={`https://releases.hashicorp.com/${name}/${version}/${shasums}`}
-          text={`SHA256 checksums for ${currentProduct.name} ${version}`}
           textSize={200}
-        />{' '}
+        >
+          {`SHA256 checksums for ${currentProduct.name} ${version}`}
```

Comment:
🚧 This no longer needs the template literal.

```suggestion
          SHA256 checksums for {currentProduct.name} {version}
```